Update IPs, Add OLT-XGS at NN1936 & NN584

bradyjoh opened the pull request:
[discussion](https://nycmesh.slack.com/archives/CAZ07JTL6/p1730303149073779?thread_ts=1729529004.373399&cid=CAZ07JTL6)

- Updating one of the existing three OLTs' IPs
- Adding XGS-OLT (hopefully their API is the same, but they're running the same firmware) units for Olmsted NN584 and 165 Broome NN1936

Andrew-Dickinson commented:
Could you attach a sample output text file? The instructions in the readme should be up to date, and running the report should be possible from anywhere inside the mesh IP space

bradyjoh commented:
Good README instructions, very easy to set up.

I encountered some issues and implemented fixes for them:
- Removed the log spam `connectionpool.py:1099: InsecureRequestWarning: Unverified HTTPS request is being made to host 'unifi.nycmesh.net'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings`
- Added error handling around missing keys for ONU name and notes. Now instead of blocking the whole report, it will log the issue and continue
- Added error handling for ConnectionTimeout exceptions, for example when an OLT's IP is wrong. Now instead of blocking the whole report, it will log the skip and continue
- Removed the 410 Grand construction hack as requested in Slack

CLI output:

```
-> % mesh-support-report
Fetching Devices Info from UFiber endpoint https://10.70.188.2/api/v1.0
Fetching Devices Info from UFiber endpoint https://10.70.188.3/api/v1.0
Fetching Devices Info from UFiber endpoint https://10.70.188.4/api/v1.0
Device Details object [{'detail': '', 'error': 1, 'message': '.wifi.networks.key: string length (0) must be between 8 and 63', 'statusCode': 400}] was missing the notes section
Device Details object [{'detail': '', 'error': 1, 'message': '.wifi.networks.key: string length (0) must be between 8 and 63', 'statusCode': 400}] was missing the name section
Device Details object [{'detail': '', 'error': 1, 'message': '.wifi.networks.key: string length (0) must be between 8 and 63', 'statusCode': 400}] was missing the notes section
Device Details object [{'detail': '', 'error': 1, 'message': '.wifi.networks.key: string length (0) must be between 8 and 63', 'statusCode': 400}] was missing the name section
Fetching Devices Info from UFiber endpoint https://10.70.188.5/api/v1.0
Fetching Devices Info from UFiber endpoint https://10.70.198.4/api/v1.0
**Daily Impaired Devices Report - Wednesday, October 30, 2024 @ 14:13**

UISP - Currently In Outage (new last 7 days)
nycmesh-189-omni (offline since 2024-10-30 @ 13:51)
nycmesh-427-omni (offline since 2024-10-30 @ 11:56)
nycmesh-lbe-4801 (offline since 2024-10-30 @ 06:40)
nycmesh-4801-omni (offline since 2024-10-26 @ 08:54)
nycmesh-303-omni / Dekalb (offline since 2024-10-25 @ 09:53)


UNIFI - Currently In Outage (new last 7 days)
Marcy_4801_West (Parks & Public places) (offline since 2024-10-30 @ 06:41)
1st floor 1 (507 - Clemente) (offline since 2024-10-28 @ 15:58)


UFIBER - Currently In Outage
410-10C (nycmesh-1932-olt3)
410-15D (nycmesh-1932-olt3)
410-15C (nycmesh-1932-olt3)
410-02A (nycmesh-1932-olt3)
410-20E (nycmesh-1932-olt3)
410-25B (nycmesh-1932-olt3)
70:a7:41:34:69:dc (nycmesh-1932-olt2)
70:a7:41:35:8e:d1 (nycmesh-1932-olt2)
460-22G (nycmesh-1932-olt2)
70:a7:41:3b:5a:28 (nycmesh-1932-olt2)
70:a7:41:3e:c4:b5 (nycmesh-1932-olt2)
460-03C (nycmesh-1932-olt2)
460-22E (doorknock no answer 04-10-2023) (nycmesh-1932-olt2)
78:45:58:9a:d4:b0 (nycmesh-1932-olt2)
460-Roof (nycmesh-1932-olt2)
460-4D? (nycmesh-1932-olt2)
d0:21:f9:a0:42:cc (nycmesh-1932-olt2)
460-07H (nycmesh-1932-olt2)
460-07C (nycmesh-1932-olt2)
131-25D (nycmesh-1934-olt1)
131-13C (nycmesh-1934-olt1)
131-12H (nycmesh-1934-olt1)
MISSING_DEVICE_NAME (nycmesh-1934-olt1)
131-09F (nycmesh-1934-olt1)
131-04B (nycmesh-1934-olt1)
70:a7:41:3b:56:e4 (nycmesh-584-olt-xgs)
West-18P-1G-Bernie (nycmesh-584-olt-xgs)
70:a7:41:3b:5c:40 (nycmesh-584-olt-xgs)
70:a7:41:3e:c5:f5 (nycmesh-584-olt-xgs)
working XGS ONU 20240820 (nycmesh-584-olt-xgs)
e4:38:83:ef:12:2e (nycmesh-584-olt-xgs)
e4:38:83:ef:b9:78 (nycmesh-584-olt-xgs)
e4:38:83:ef:bc:03 (nycmesh-584-olt-xgs)
e4:38:83:ef:bc:2a (nycmesh-584-olt-xgs)
East-15B-1G-Erik (nycmesh-584-olt-xgs)


UFIBER - Poor Experience (< 100%)
78:45:58:9a:c7:e8 (nycmesh-1932-olt3) (80%)
460-19B (nycmesh-1932-olt2) (80%)
460-12F (nycmesh-1932-olt2) (77%)
MISSING_DEVICE_NAME (nycmesh-1934-olt1) (80%)


UFIBER - Poor Signal (< -28 dBm)
-- None --


Ignored Issues - All Sources
nycmesh-3461-af60lr-5916 (maintenance mode)
nycmesh-527-west (maintenance mode)
410-08C (unplugs some times)
131-07H (unplugs sometimes)

Not uploading to slack. Set the environment variable SLACK_ENABLED=True to enable automatic posting to slack

```

Andrew-Dickinson commented on src/mesh_support_report_generator/ufiber_outages.py:
```diff
@@ -15,6 +17,10 @@
 
 
 def login(session: requests.Session, ufiber_base_url: str):
+    # Suppress the 'Unverified HTTPS request is being made to host' log spam
+    # because the POST is made with verify=False
+    requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```
You just taught we something new, thanks!
